### PR TITLE
[#55] Update README.md

### LINKED ISSUE #55: Update README.md
## Why

Update `README.md` with clear instructions for future developers

## Acceptance Criteria

- Provide staging and production URLs
- Specify requirements to run the app locally (ex: environment variables, versions, ...)

## Design
N/A

## Resources
N/A

### PR DESCRIPTION
**Pull request by toby-thanathip**
- Close #55 

## What happened 👀

Update README.md with relevant information

## Proof Of Work 📹

**PREVIEW**

Screen Shot 2023-07-20 at 14 10 06

**POSTMAN COLLECTION**

Screen Shot 2023-07-20 at 14 10 38

### CODE REVIEW

**Comment by Tuubz** [comment]
@olivierobert @hoangmirs @hoangnguyen92dn Final one (besides the 0.7.0 release PR) 🎉

**Comment by Tuubz** [comment]
TODO: Add endpoint from #57 to the postman collection

File: README.md [hunk 1]
```diff
@@ -1,9 +1,54 @@
-[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+Staging: https://ic-rails-huey-toby-staging.herokuapp.com/
```
Comment (olivierobert) [inline README.md]: Minor improvement idea to center all this content and use badges for everything:

<p align="center">
  
  <a href="https://ic-rails-huey-toby-staging.herokuapp.com/">
     
  </a>
  <a href="https://ic-rails-huey-toby.herokuapp.com/">
     
  </a>
</p>

**Review by olivierobert — COMMENTED** [review]
Did you forget to replace the placeholder content? 🙈 

Screenshot 2566-07-20 at 05 07 49

File: README.md [hunk 2]
```diff
@@ -1,9 +1,54 @@
...
+2. Set the correct environment variables in `application.yml`
+3. Start Rails server: `make dev`
+4. Start Sidekiq: `bundle exec sidekiq`
+
+## Usage
+
+Here's a summary of the available endpoints:
+
+- Sign-up with e-mail/password and Google.
+- Sign-in wit e-mail/password and Google.
```
Comment (hoangnguyen92dn) [inline README.md]:
```
- Sign-in with e-mail/password and Google.
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,54 @@
...
+- Upload keywords with Google/Bing.
+- Show list of search results.
+- Filter through search results.
+- Show details of search result.
+- Download PDF of search result.
+
+For convenience, you can import the Postman collection: `postman-collection.json`
```
Comment (hoangnguyen92dn) [inline README.md]: How about adding the link to the collection so readers can easily find it 😄

File: postman-collection.json [hunk 4]
```diff
@@ -0,0 +1,1119 @@
...
+						},
+						"method": "GET",
+						"header": [],
+						"body": {
+							"mode": "formdata",
+							"formdata": [
+								{
+									"key": "csv_file",
+									"type": "file",
+									"src": "/Users/tuubz/Desktop/keywords.csv"
```
Comment (hoangnguyen92dn) [inline postman-collection.json]: Would it be possible to revise this? 🤔

File: postman-collection.json [hunk 5]
```diff
@@ -0,0 +1,1119 @@
...
+								{
+									"key": "client_secret",
+									"value": "{{CLIENT_SECRET}}",
+									"type": "text",
+									"disabled": true
+								}
+							]
+						},
+						"url": {
+							"raw": "{{base_url}}/api/v1/search_results?filter[url_contains_at_least_two]=:",
```
Comment (hoangnguyen92dn) [inline postman-collection.json]: Following Olivier's suggestion https://github.com/nimblehq/ic-rails-huey-toby/pull/57#discussion_r1268879840, I updated the endpoint for more flexible, please help to update as well 😅

**Review by olivierobert — APPROVED** [review]

**Review by hoangmirs — APPROVED** [review]

**Review by hoangnguyen92dn — APPROVED** [review]

**Pull request merged**